packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -1209,9 +1221,74 @@ mixin WidgetsBinding on BindingBase, ServicesBinding, SchedulerBinding, GestureB
 ///    ensure the widget, element, and render trees are all built.
 void runApp(Widget app) {
   final WidgetsBinding binding = WidgetsFlutterBinding.ensureInitialized();
-  assert(binding.debugCheckZone('runApp'));
+  _runWidget(binding.wrapWithDefaultView(app), binding, 'runApp');
+}
+
+/// Inflate the given widget and bootstraps the widget tree.
```

Comment:
```suggestion
/// Inflates the given widget and bootstraps the widget tree.
```

Reply:
To match the existing docs on `runApp` I am going to change this to:

```suggestion
/// Inflate the given widget and bootstrap the widget tree.
```